### PR TITLE
Two processes fix#7 WIP

### PR DESCRIPTION
**Pull request by 1v4n4**
#7

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+
+Lookup returns:
 
-## Lookup
+- true if email in database
+- false when not found
+- error if anything goes wrong
 
--
-- The request payload should be `{email: "some@email.com"}`
```
Comment (tttp) [inline README.MD:10]: leave the format of the request, or is this with upstream/trust and not proca-server->lookup?

Comment (1v4n4) [reply README.MD:10]: That is for the old lookup (to get status  from LC)

File: src/http.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  try {
+    const email = request.query.email;
+    const isSubscribed = await lookup(email);
+    return formatResult (isSubscribed);
+  } catch (error) {
+    console.log(error);
+    reply.code(500).send({ isError:true,error: error.toString(),found:false });
+  }
+});
+
```
Comment (1v4n4) [inline src/http.ts:76]: Should we remove what we do not use? I think only two posts (for update and lookup are needed

File: bin/remotelookup.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const http = require("http");
+const axios = require("axios");
+
```
Comment (1v4n4) [inline bin/remotelookup.js:5]: This file is the same as testlookup.js, neither does remote (trust) lookup. What is this for? And where's remote lookup?

**Comment by 1v4n4** [comment]
@tttp check the comments, please

File: bin/remotelookup.js (hunk 3 above)
Comment (tttp) [reply bin/remotelookup.js:5]: they are calling different endpoints

remote lookup is calling trust/lookup that is doing the remote lookup

File: src/http.ts (hunk 2 above)
Comment (tttp) [reply src/http.ts:76]: post lookup is the real lookup to use, so very much needed.

as for the try catch, why are you assuming it's not called? In general, it's good practice and the polite way to say to plan for the unplan and return an error to the called

**Pull request merged**